modules/i18n/src/translations/en-US/portals/actions.ts

```diff
@@ -192,10 +196,14 @@ export const actions: actionsNS = {
                 description: "Couldn't delete the action.",
                 message: "Something went wrong."
             },
-            fetch: {
+            fetchById: {
                 description: "Couldn't fetch the action.",
                 message: "Something went wrong."
             },
+            fetchByType: {
+                description: "Couldn't fetch actions.",
```

Comment:
Let's use user friendly wordings here.
```suggestion
                description: "Couldn't retrieve actions.",
```